subsys/net/lib/config/init.c

```diff
@@ -553,17 +553,8 @@ int net_config_init_app(const struct device *dev, const char *app_info)
 	/* This is activated late as it requires the network stack to be up
```

Comment:
Typo in commit `net: config: init: remove second init of syslog`
s/durring/during/